src/main/scala/com/gd/twitterstreamingtToDatalake/StreamingTweetsJob.scala

```diff
@@ -0,0 +1,41 @@
+package com.gd.twitterstreamingtToDatalake
+
+import org.apache.log4j.Logger
+
+object StreamingTweetsJob extends SparkSessionWrapper {
+
+  //Set the Logger Level
+  val log = Logger.getLogger(StreamingTweetsJob.getClass.getName)
+
+  def main(args: Array[String]): Unit = {
+
+    if (validateConfiguration(args)) {
+      System.exit(1)
+    }
+    val arr = args.take(4)
+    TweetsIngestion.configureTwitter(arr)
+
+    //Incase Only tweets related to a specific Hashtags are needed..
+    val filters = args.takeRight(args.length - 4)
+
+    val englishTweets = TweetsIngestion.getTweets(filters)
+    val text = TransformTweets.getText(englishTweets)
+    val hashTags = TransformTweets.getHashTags(text)
+
+    hashTags.saveAsTextFiles("tweets", "json")
+
+    ssc.start()
+    ssc.awaitTermination()
+  }
+
+  def validateConfiguration(args: Array[String]): Boolean = {
+
+    if (args.length < 4) {
```

Comment:
Since there is no specific pattern for twitter keys and that is just random keys, I have added a function which can check the pattern in all keys, if given. Currently it just checks the empty string.